warp/src/tesseract/mod.rs

```diff
@@ -478,29 +499,96 @@ impl Tesseract {
     pub fn lock(&self) {
         self.inner.lock();
     }
+}
 
-    /// To save to file using internal file path.
-    /// Note: Because we do not want to interrupt the functions due to it failing to
-    ///       save for whatever reason, this function will return `Result::Ok`
-    ///       regardless of success or error but would eventually log the error.
-    ///
-    /// TODO: Handle error without subjecting function to `Result::Err`
-    pub fn save(&self) -> Result<()> {
-        self.inner.save()
+/// Methods for wasm only
+#[cfg(target_arch = "wasm32")]
```

Comment:
```suggestion
```

Reply:
We do want to keep this here, or the compiler will complain on non-wasm that `wasm_bindgen` is not imported. It is only imported on wasm.

Reply:
This shouldnt be needed since we have `#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]`, though we could probably drop this and use `#[cfg(target_arch = "wasm32")]`. Thoughts?